example/FluidMotion/dart/FluidMotion.dart

```diff
@@ -1,9 +1,11 @@
+import 'dart:html';
+
 /// Copyright 2013 the V8 project authors. All rights reserved.
 /// Copyright 2009 Oliver Hunt <http://nerget.com>
 ///
 /// Permission is hereby granted, free of charge, to any person
 /// obtaining a copy of this software and associated documentation
-/// files (the "Software"), to deal in the Software without
```

Comment:
These shouldn't have been caught by the lint? Is the dartfix approach changing them anyway?

Reply:
This was likely done by my regex logic...